pkg/operator/controller/ingress/controller_test.go

```diff
@@ -1,16 +1,14 @@
 package ingress
 
 import (
+	"github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"reflect"
 	"testing"
 	"time"
 
-	"github.com/google/go-cmp/cmp"
-
 	configv1 "github.com/openshift/api/config/v1"
 	operatorv1 "github.com/openshift/api/operator/v1"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Did your editor do this?  This is an odd re-ordering.

Reply:
yeah the editor does it